bin/curi_builder/builder/__init__.py

```diff
@@ -12,22 +12,23 @@
 
 
 def find_changed(sha):
-    r = subprocess.run(
-        ["git", "--no-pager", "diff", sha, "--name-only", "./deployments"], stdout=subprocess.PIPE
-    )
-    s = r.stdout.decode("utf-8").split("\n")[:-1]
+    directory_to_watch = ["./deployments","./jobs"]
+    changes_list  = []
+
+    for dir in directory_to_watch:
+        completed_process =  subprocess.run(
+            ["git", "--no-pager", "diff", sha, "--name-only", dir], stdout=subprocess.PIPE
+        )
+        changes_list += completed_process.stdout.decode("utf-8").split("\n")[:-1]
 
     return [
         {
-            "path": f"./{'/'.join(x.split('/')[:-2])}",
-            "deployment": x.split("/")[1],
-            "service": x.replace("/services/pulse3d", "/services/pulse3d_api").split("/")[
-                3
-            ],  # leaving services to prevent switching for pulse3d-worker
+            "path": f"./{'/'.join(ch.split('/')[:-2])}",
+            "deployment": ch.split("/")[1],
+            "service": "pulse3d_api" if "deployments" in ch else "pulse3d-worker"
```

Comment:
this won't work, there are other services in deployments besides pulse3d_api and in the future there will be other jobs besides pulse3d-worker